src/colvargrid.h

```diff
@@ -1606,6 +1861,63 @@ class integrate_potential : public colvar_grid_scalar
     add_constant(-1.0 * minimum_value());
   }
 
+  /// \brief Return the value at the upper edge/corner of the given bin,
+  /// linearly interpolated (averaged) between 2^nd neighboring bins
+  /// Used to interpolate to a grid shifted by a half-bin
+  /// e.g. from PMF to gradient grid
+  /// should not be used on upper edges of non-periodic grids
+  /// (that should never be necessary, since those are extra bins with respect
+  /// to underlying gradient grid)
+  inline cvm::real value_corner_interp(std::vector<int> const &ix,
```

Comment:
Do you think this could be moved to the class template? I'd like to use it as a basis for a function to interpolate intermediate values of a grid for a more continuous metadynamics potential.

Reply:
Turns out this code is unused in the current version, it was used in an experimental feature that got pruned. But a more general linear interpolation function would be very useful, and this combined with current_offset would do the trick.